cmd/wgcf-cli/main.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const ConfigPathDefault string = "wgcf.json"
+
```

Comment:
Why?

Reply:
This is used in main_test.go

If one literal value with same meaning used in several places, then creating one explicit constant variable recommended for this purpose.